scripts/quests/otherAreas/For_the_Birds.lua

```diff
@@ -0,0 +1,211 @@
+-----------------------------------
+-- For the Birds
+-----------------------------------
+-- Log ID: 4, Quest ID: 104
+-- Koblakiq          !pos -64 21 -117 11
+-- Daa Bola the Seer !pos -157 -17 193 151
+-- GeFhu Yagudoeye   !pos -91 -3 -127 147
+-----------------------------------
+local oztrojaID  = zones[xi.zone.CASTLE_OZTROJA]
+local beadeauxID = zones[xi.zone.BEADEAUX]
+-----------------------------------
+local quest = Quest:new(xi.questLog.OTHER_AREAS, xi.quest.id.otherAreas.FOR_THE_BIRDS)
+
+quest.reward =
+{
+    item = xi.item.JAGUAR_MANTLE,
+}
+
+local mobListTable =
+{
+    beadeauxID.mob.FOR_THE_BIRDS_MOB_OFFSET,
+    beadeauxID.mob.FOR_THE_BIRDS_MOB_OFFSET + 1,
+    beadeauxID.mob.FOR_THE_BIRDS_MOB_OFFSET + 2,
+    beadeauxID.mob.FOR_THE_BIRDS_MOB_OFFSET + 3,
+}
+
+local function AllDead(mobs)
```

Comment:
Please start local functions with a lowercase, only globals should use upper

Reply:
Done ✅